Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -611,3 +611,13 @@ public struct ReplaceTokensFixIt: ParserFixIt {
     "replace \(nodesDescription(replaceTokens, format: false)) with '\(replacement.text)'"
   }
 }
+
+public struct ReplaceWithMultipleTokensFixIt: ParserFixIt {
+  public let replaceTokens: [TokenSyntax]
+  
+  public let replacements: [TokenSyntax]
+  
+  public var message: String {
+    "replace \(nodesDescription(replaceTokens, format: false)) with \(nodesDescription(replacements, format: false))"
+  }
+}
```

Comment:
Instead of introducing a new Fix-It you could just generalize the existing `ReplaceTokensFixIt` to allow multiple tokens for `replacement`.

Reply:
You’ve hereby been granted permission 😄